src/gui/src/settings/options-window.h

```diff
@@ -123,7 +122,7 @@ class OptionsWindow : public QDialog
 		QList<QLineEdit*> m_customNames, m_customTags, m_filenamesConditions, m_filenamesFilenames, m_filenamesFolders;
 		QList<TokenSettingsWidget*> m_tokenSettings;
 		QList<QPair<QLineEdit*, QLineEdit*>> m_metadataPropsys, m_metadataExiftool;
-		QList<std::pair<QCheckBox*, QSpinBox*>> csPairs;
+		QList<QPair<QCheckBox*, QSpinBox*>> m_zoomSettingPairs;
```

Comment:
I agree that csPairs is a bad name, but this container would ideally also be used for buttons in other windows. Could we call it something like "m_buttonSettingPairs"? This doesn't actually contain the ButtonSetting objects though.

I've seen this notation in other parts of the code-- what does the "m_" mean?

Reply:
You can rename it to something else.

> I agree that csPairs is a bad name, but this container would ideally also be used for buttons in other windows. Could we call it something like "m_buttonSettingPairs"?

Wouldn't you use a separate list for each window? Otherwise when checking for duplicate values or saving the values in QSettings you wouldn't know what goes where.

> I've seen this notation in other parts of the code-- what does the "m_" mean?

"Member". It allows to know that the variable is a private/protected member when accessing it in the code. It's quite common to see `g_` for globals and `s_` for static.

Reply:
I see. Good to know.

The way I imagine using this for other buttons would be populating it selectively, for example when a page or tab in the options window is loaded. The problem with having a different one for each window is that it increases resource utilisation for no reason, since users will probably only configure one window's buttons at a time. If they are somehow configuring buttons for multiple windows at the same time (which would probably be a bad U.I.), I would probably change the structure of this data member. It's up to you though, I don't plan to extend the button functionality for other windows any time soon.